administrator-guides/message-auditing/README.md

```diff
@@ -0,0 +1,74 @@
+# Message Auditing Panel
```

Comment:
Can you add this file to the [table of contents](https://github.com/RocketChat/docs/blob/master/_data/toc.yml)?

Reply:
Updated TOC.